### DISCUSSION TITLE
Workflow notification to author

### DISCUSSION THREAD

**Discussion by stimpsonjcat** [joomla/joomla-cms#38718] [2. Cold Features]
Workflows sends notifications to usergroups or to specific defined users. It would be nice if a workflow transition can send a notification to the author of the article. Now you can only notify _all_ authors.

**Usecase**: an author writes an article, but it needs to be checked by an editor before it's published. If the editor approves the article, it would be nice if the author is notified the article is published or if the article is rejected the author could be notified as well, without letting all users in the author group te be notified.

**Comment by chmst** [comment]
Thank you for this it is a good idea. 
There is even already a feature request  https://github.com/joomla/joomla-cms/issues/38008.
